src/pkg/selectors/sharepoint.go

```diff
@@ -333,6 +335,46 @@ func (s *sharePoint) PageItems(pages, items []string, opts ...option) []SharePoi
 // -------------------
 // Filter Factories
 
+func (s *sharePoint) CreatedAfter(timeStrings string) []SharePointScope {
+	return []SharePointScope{
+		makeFilterScope[SharePointScope](
+			SharePointLibraryItem,
+			FileFilterCreatedAfter,
```

Comment:
You added enums for `SiteFilter...`, but I only see `FileFilter...` here in the scope creation.